### PR TITLE
[DEV-152] Add Philippines affiliate doc

### PR DESCRIPTION
**Pull request by randelpalu**

### CODE REVIEW

File: affiliates/philippines.swagger.json [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+      "version" : "3.0.0"
+    },
+    "servers" : [ {
+      "url" : "http://api.staging.credy.eu/v3",
```
Comment (mikk150) [inline affiliates/philippines.swagger.json]: if you want to specify servers, you can do that, but then you have to add dash at the end of server declaration, and remove slashes at the beginning of all paths

File: affiliates/philippines.swagger.json [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+            "name" : "Accept-Language",
+            "in" : "header",
+            "description" : "To get response error messages in filipino, header’s value must be set to according ISO code. Ex: `fil` or `fil-PH`.",
+            "required" : false,
+            "style" : "simple",
+            "explode" : false,
+            "schema" : {
+              "format" : "string"
+            },
+            "example" : "fil-PH"
```
Comment (mikk150) [inline affiliates/philippines.swagger.json]: we actually do not support `fil-PH` language

File: affiliates/philippines.swagger.json (hunk 1 above)
Comment (randelpalu) [reply affiliates/philippines.swagger.json]: adjusted

File: affiliates/philippines.swagger.json (hunk 2 above)
Comment (randelpalu) [reply affiliates/philippines.swagger.json]: removed this

File: affiliates/philippines.swagger.json [hunk 3]
```diff
@@ -0,0 +1,479 @@
...
+      "oAuth": []
+    }],
+    "paths" : {
+      "customers" : {
+        "post" : {
+          "description" : "Register customer",
+          "parameters" : [ {
+            "name" : "Accept-Language",
+            "in" : "header",
+            "description" : "To get response error messages in filipino, header’s value must be set to according ISO code. Ex: `fil` or `fil-PH`.",
```
Comment (mikk150) [inline affiliates/philippines.swagger.json]: we do not support fil-PH

File: affiliates/philippines.swagger.json [hunk 4]
```diff
@@ -0,0 +1,479 @@
...
...If you have any questions, please do not hesitate to contact the IT manager at [it@credy.eu](mailto:it@credy.eu).\n\n## General information\n\nProduction URL: [https://api.credy.eu/v3/](https://api.credy.eu/v3/)\n\nStaging URL: [http://api.staging.credy.eu/v3/](http://api.staging.credy.eu/v3/)\n\nAPI supports post fields, XML and JSON.\n\nTo get response error messages in filipino, \"Accept-Language\" header’s value must be set to according ISO code. Ex: `fil` or `fil-PH`.\n## Authorization\nAuthorization is performed with OAuth 2.0 Client Credentials Grant.\nClient key and secret key will be assigned to you by Credy.\n## Lead registration flow\n\n1. Request is sent to /v3/customers\n\n2. If request is successful, customer uuid must be stored\n\n3. Request is sent to /v3/leads with previously stored customer uuid in query parameter \"customer\" *ex. /v3/leads?customer=a3ff7c20-e9e8-43a0-bfb7-71a5cf43b8d3* \n4. When the lead is accepted by api (under_investigation is received from POST lead), please redirect the customer to [https://www.credy.vn/loading/?lead={UUID}](https://www.credy.vn/loading/?lead={UUID}) (where {UUID} is replaced by the uuid that you receive from POST lead)\n",
```
Comment (mikk150) [inline affiliates/philippines.swagger.json]: we do not support fil-ph

File: affiliates/philippines.swagger.json [hunk 5]
```diff
@@ -0,0 +1,479 @@
...
+      "description" : "Staging server"
+    }, {
+      "url" : "https://api.credy.eu/v3/",
+      "description" : "Production server"
+    } ],
+    "security": [{
+      "oAuth": []
+    }],
+    "paths" : {
+      "customers" : {
```
Comment (mikk150) [inline affiliates/philippines.swagger.json]: interestingly, slash still has to be in front of it

File: affiliates/philippines.swagger.json [hunk 6]
```diff
@@ -0,0 +1,479 @@
...
+              }
+            }
+          }
+        }
+      },
+      "parameters" : {
+        "AcceptLanguageHeader" : {
+          "name" : "Accept-Language",
+          "in" : "header",
+          "description" : "To get response error messages in filipino, header’s value must be set to according ISO code. Ex: `fil` or `fil-PH`.",
```
Comment (mikk150) [inline affiliates/philippines.swagger.json]: we do not support fil-PH

File: affiliates/philippines.swagger.json [hunk 7]
```diff
@@ -0,0 +1,479 @@
...
+          "required" : false,
+          "style" : "simple",
+          "explode" : false,
+          "schema" : {
+            "format" : "string"
+          },
+          "example" : "fil-PH"
```
Comment (mikk150) [inline affiliates/philippines.swagger.json]: we do not support fil-PH

File: affiliates/philippines.swagger.json [hunk 8]
```diff
@@ -0,0 +1,479 @@
...
+            "name" : "Accept-Language",
+            "in" : "header",
+            "description" : "To get response error messages in filipino, header’s value must be set to according ISO code. Ex: `fil` or `fil-PH`.",
+            "required" : false,
+            "style" : "simple",
+            "explode" : false,
+            "schema" : {
+              "format" : "string"
+            },
+            "example" : "fil-PH"
```
Comment (mikk150) [inline affiliates/philippines.swagger.json]: we do not support fil-PH

**Review by mikk150 — CHANGES_REQUESTED** [review]

File: affiliates/philippines.swagger.json (hunk 8 above)
Comment (randelpalu) [reply affiliates/philippines.swagger.json]: fixed

**Comment by mikk150** [comment]
LGTM

**Review by mikk150 — APPROVED** [review]

**Comment by Nuffic** [comment]
image

The redirect url is invalid

**Comment by randelpalu** [comment]
@Nuffic 
changed redirect url to .ph

**Pull request merged**